simple-backend/nlpviewer_backend/handlers/project.py

```diff
@@ -0,0 +1,75 @@
+from django.contrib import admin
+from django.urls import include, path
+from django.http import HttpResponse, JsonResponse
+from django.forms import model_to_dict
+import uuid
+import json
+from ..models import Project, Document, User
+from ..lib.require_login import require_login
+
+
+@require_login
+def listAll(request):
+    projects = Project.objects.all().values()
+    return JsonResponse(list(projects), safe=False)
+
+
+@require_login
+def create(request):
+    received_json_data = json.loads(request.body)
+
+    project = Project(
+        name=received_json_data.get('name'),
+        # documents=received_json_data.get('documents'),
+        ontology=received_json_data.get('ontology')
+    )
+
+    project.save()
+
+    return JsonResponse({"id": project.id}, safe=False)
+
+
+@require_login
+def edit(request, project_id):
+    pro = Project.objects.get(pk=project_id)
```

Comment:
just use full name "project", no need to use abbreviations here.

Reply:
fixed